app/src/main/java/org/exoplatform/activity/AddDomainServerActivity.java

```diff
@@ -76,7 +79,46 @@ public boolean onTouch(View arg0, MotionEvent arg1) {
                 return false;
             }
         });
+        companyTextField.addTextChangedListener(new TextWatcher() {
 
+            @Override
+            public void onTextChanged(CharSequence arg0, int arg1, int arg2, int arg3) {
+
+            }
+            @Override
+            public void beforeTextChanged(CharSequence arg0, int arg1, int arg2,
+                                          int arg3) {
+            }
+            @Override
+            public void afterTextChanged(Editable et) {
+                String s=et.toString();
+                if(!s.equals(s.toLowerCase())){
+                    s=s.toLowerCase();
+                    companyTextField.setText(s);
+                    companyTextField.setSelection(companyTextField.length()); //fix reverse texting
+                }
```

Comment:
```suggestion
                companyTextField.setText(et.toString().toLowerCase());
                companyTextField.setSelection(companyTextField.length()); //fix reverse texting
```

Reply:
I have to prevent keyboard from writing Uppercase characters while editing